src/components/SelectionList/OnboardingListItem.tsx

```diff
@@ -0,0 +1,105 @@
+import {Str} from 'expensify-common';
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import Text from '@components/Text';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import BaseListItem from './BaseListItem';
+import type {ListItem, OnboardingListItemProps} from './types';
+
+function OnboardingListItem<TItem extends ListItem>({
```

Comment:
Can't we just use `RadioListItem`?

Reply:
> We use 32px padding on desktop modals and 20px on mobile :)

Because we need the custom padding for desktop and mobile then I created a new component.

Reply:
Using a new component here seems an overkill. The difference is just in the padding value, can you instead pass that as a prop?

Reply:
> Using a new component here seems an overkill. The difference is just in the padding value, can you instead pass that as a prop? 

@s77rt 
We don't have any prop to do that now then we need to introduce a new prop. Instead, we can use `RadioListItem` with `leftElement` with `marigin-left` as `12` for desktop and `0` for mobile and add the same `margin-right` for `rightElement`. What do you think?

Reply:
If the row will still take full width (e.g. on hover) then let's do it

Reply:
> If the row will still take full width (e.g. on hover) then let's do it

It take full width, updated.